include/function2/function2.hpp

```diff
@@ -1828,6 +1828,16 @@ constexpr auto overload(T&&... callables) {
 }
 } // namespace fu2
 
+namespace std{
+template <typename Alloc,
+  typename Config, bool IsThrowing, bool HasStrongExceptGuarantee,
+  typename... Args>
+struct uses_allocator<
+  ::fu2::detail::function<Config, ::fu2::detail::property<IsThrowing, HasStrongExceptGuarantee, Args...>>,
```

Comment:
Like `Config` you do not have to specialize property, a simple `typename Property` is sufficient and will reduce the specialization to `::fu2::detail::function<Config, Property>`.

Reply:
Well, I just followed your comment in https://github.com/Naios/function2/issues/61#issuecomment-1962543547 . I thought you want it more specific. I've made the change in new commit.